### DISCUSSION TITLE
"Error resolving url" when insert a link

### DISCUSSION THREAD

**Discussion by alripica** [mozilla/hubs#4986] [Hubs Cloud Questions]
When I insert the link on hubs.mozilla.com it works fine, but it not works on AWS.
When I insert the image, both work fine.

at the link below, it seems that it has already been updated.
[https://github.com/mozilla/hubs/discussions/3652#discussioncomment-263747]()

I had subscribed Personal v1.1.2 on AWS.
Do I need additional setup to insert external links?

![image](https://user-images.githubusercontent.com/6283893/147805470-3310de31-6b3e-4ad9-8f58-9b5bd58802eb.png)

![image](https://user-images.githubusercontent.com/6283893/147805466-da605d06-db7c-459c-9236-b49fc7d1a004.png)

![image](https://user-images.githubusercontent.com/6283893/147805463-89ad07da-9873-453e-bb6f-63a26f3c0311.png)

![image](https://user-images.githubusercontent.com/6283893/147805456-22192ad8-ea68-456a-9884-2e9d6c341131.png)

**Comment by pattersonbl2** [comment]
By chance, do you have to link you want to post I want to test in my own hubs instance ?
